tools/make/kube.mk

```diff
@@ -1,7 +1,7 @@
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION ?= 1.28.0
 # Need run cel validation across multiple versions of k8s
-ENVTEST_K8S_VERSIONS ?= 1.27.1 1.28.0 1.29.0
+ENVTEST_K8S_VERSIONS ?= 1.28.0 1.29.0 1.30.0 1.31.0
```

Comment:
can we pick the latest patch versions here ?
```
1.28.13 1.29.8 1.30.4 1.31.0
```

Reply:
I tried(but failed), it's actual `kubebuilder` versions.

Reply:
ah thanks for sharing that link, here's what I see in the envtest releases
```
1.28.3 1.29.5 1.30.3 1.31.0
```